pkg/agent/util/nftables/nftables.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nftables
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/knftables"
+)
+
+const table = "antrea"
+
+type Client struct {
+	// It's convenient to use the fake nftables instance when these two fields are capitalized outside the package.
```

Comment:
I don't think you need a justification for why the 2 fields are public. It is useful beyond just testing. Sometimes it is more convenient to access the fields directly instead of using `All` (and `All` creates a map...). I see some occurrences in `route_linux.go` where you access these fields directly.